istioctl/cmd/auth.go

```diff
@@ -102,41 +102,47 @@ THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 		},
 	}
 
-	upgradeCmd = &cobra.Command{
-		Hidden: true,
-		Use:    "upgrade",
-		Short:  "Upgrade Istio Authorization Policy from version v1alpha1 to v1beta1",
-		Long: `Upgrade converts Istio authorization policy from version v1alpha1 to v1beta1. It requires access to Kubernetes
-service definition in order to translate the service name specified in the ServiceRole to the corresponding
-workload labels in the AuthorizationPolicy. The service definition could be provided either from the current
-Kubernetes cluster or from a YAML file specified from command line.
+	convertCmd = &cobra.Command{
+		Use:   "convert",
```

Comment:
convert or migrate? I think this is different from the istio.io PR.

Reply:
I changed to use the term `convert` for 2 reasons: 1) align with the other `ingress convert` command in istioctl, the `ingress convert` converts k8s ingress resource to Istio ingress; 2) `upgrade` sometimes mean the end-to-end upgrade including both convert and apply, which is not the reality of this command.

Reply:
sgtm, what I meant was the PR from istio.io iirc uses `istioctl x migrate`, so we should update that (or this, but seems like you're leaning toward `convert`).